students/ScottL/lesson04/mailroom_part2.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+#-------------------------------------------------#
+# Title: Mailroom Part-2 Assignment
+# Dev: Scott Luse
+# Date: Feb 10, 2018
+#-------------------------------------------------#
+
+
+
+# -- Data --#
+DONOR_HISTORY_DICT = {
+    'Peter Parker': [288.09, 9.01, 61288.09],
+    'Iron Man': [1238.09, 8199.01, 1468.07],
+    'Captain Marvel': [43188.09, 1288.09],
+    'Black Widow' : [1272.09],
+}
+
+# -- Processing --#
+def GetUserChoice():
+
+    print("""
+    MailRoom Part2 Menu Options
+    1) Send a Thank You
+    2) Create a Report
+    3) Send Letters To Everyone
+    4) Quit Program
+    """)
+    user_choice = input("Which option would you like to perform? [1 to 4]: ")
+    return (user_choice.strip())
+
+
+def process_menu(menu_item):
+    if (menu_item == '1'):
+        send_thanks()
+        return
+    elif (menu_item == '2'):
+        format_charity_structure()
+        return
+    elif (menu_item == '3'):
+        create_individual_letters()
+        return
+
+def send_thanks():
+    while (True):
+        thank_you_name = input("Please enter full name, enter 'list' for names, or 'main' for menu: ")
+        if (thank_you_name.lower() == "list"):
+            print_donors()
+            print(' ')
+        elif (thank_you_name.lower() == "main"):
+            return
+        else:
+            process_input_name(thank_you_name.title())
```

Comment:
this `.title()` doesn't give you anything more than just using the variable name `thank_you_name`